src/view.options.js

```diff
@@ -28,65 +35,78 @@ class OptionsView {
   _load() {
     this._eachOption(
       ($elm, key, value, cb) => {
-        if ($elm.is(':checkbox')) $elm.prop('checked', value);
+        const checkboxElement = $elm.is(':checkbox')
+
+        if (checkboxElement) $elm.prop('checked', value);
         else $elm.val(value);
+
+        // Attaches (once) on value change event
+        if (!$elm.data('octotree-on-change-event-attached')) {
+          $elm.data('octotree-on-change-event-attached', true);
+          const boundOnChange = this._saveOption.bind(this, $elm, key);
```

Comment:
Why not save only when users switch to a different view?
Would be simpler in logic. And reuse current saveOptions().